guides/hack/56-modules/06-type-aliases.md

```diff
@@ -0,0 +1,44 @@
+Type aliases have special rules when it comes to internal types depending on which kind of type alias is used. 
+
+## Internal type aliases
+An opaque or normal type alias can be marked internal. Just as with classes, this means that the type can only be referenced from within the module.
```

Comment:
I'd use concrete syntax rather than saying 'opaque or normal', as you're referring to a single kind of type type alias.

```suggestion
A type alias declared with `type` can be marked internal. Just as with classes, this means that the type can only be referenced from within the module.
```